src/main/groovy/wooga/gradle/build/unity/tasks/AbstractUnityBuildEngineTask.groovy

```diff
@@ -0,0 +1,161 @@
+package wooga.gradle.build.unity.tasks
+
+
+import org.gradle.api.file.Directory
+import org.gradle.api.file.RegularFile
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Optional
+import wooga.gradle.build.unity.internal.BuildEngineArgs
+import wooga.gradle.secrets.internal.Secrets
+import wooga.gradle.unity.UnityTask
+
+import javax.crypto.spec.SecretKeySpec
+
+
+abstract class AbstractUnityBuildEngineTask extends UnityTask {
+
+    private final Property<String> build
+    private final Property<String> version

```

Comment:
I am undecided on the version being here since it's required only by the PlayerBuildRequest

Reply:
It isn't, the version is being required by the base task as well.

Reply:
Removed this property, is it isn't needed for basic builds.